config/settings.defaults.coffee

```diff
@@ -38,6 +38,7 @@ settings =
 				key: process.env['AWS_ACCESS_KEY_ID']
 				secret: process.env['AWS_SECRET_ACCESS_KEY']
 				endpoint: process.env['AWS_S3_ENDPOINT']
+				partSize: process.env['S3_PARTSIZE'] or (100 * 1024 * 1024)
```

Comment:
```suggestion
				partSize: process.env['AWS_S3_PARTSIZE'] or (100 * 1024 * 1024)
```
Let's be consistent with the other environment variables and prefix it with `AWS_`.